packages/core/src/browser/storage-service.ts

```diff
@@ -43,9 +44,12 @@ interface LocalStorage {
 @injectable()
 export class LocalStorageService implements StorageService {
     private storage: LocalStorage;
+    private fullstorage: boolean = false;
```

Comment:
why coping the state into a field?

Reply:
We try to copy the data to the local storage only if there is still some room. Once we reach the quota, no need to try to write to the local storage anymore. The flag is to maintain the value if the user has previously reach the maximun.

Reply:
If we keep this field, storageFull may be a better name